internal/cmd/ske/credentials/complete-rotation/complete_rotation.go

```diff
@@ -0,0 +1,117 @@
+package completerotation
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/ske/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/spinner"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske/wait"
+)
+
+const (
+	clusterNameArg = "CLUSTER_NAME"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ClusterName string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("complete-rotation %s", clusterNameArg),
+		Short: "Completes the rotation of the credentials associated to a SKE cluster",
+		Long: fmt.Sprintf("%s\n%s\n%s\n%s\n%s\n\n%s\n%s\n%s\n%s",
+			"Completes the rotation of the credentials associated to a STACKIT Kubernetes Engine (SKE) cluster.",
+			"To ensure continued access to the Kubernetes cluster, please update your kubeconfig service account to the newly created account.",
+			"This is step 1 of a 2-step process to rotate all SKE cluster credentials. Tasks accomplished in this phase include:",
+			"  - The old certification authority will be dropped from the package.",
+			"  - The old signing key for the service account will be dropped from the bundle.",
+			"If you haven't, please start the process by running:",
+			"  $ stackit ske credentials start-rotation my-cluster",
+			"Generate a new kubeconfig file by running:",
+			"  $ stackit ske kubeconfig create my-cluster"),
```

Comment:
Is this meant like a follow-up tip? Maybe we could emphasize that this can be done after the complete rotation, e.g. "After completing the rotation, you can generate ...`